script/translations/download.py

```diff
@@ -56,35 +55,39 @@ def run_download_docker():
         raise ExitApp("Failed to download translations")
 
 
-def save_json(filename: str, data: list | dict):
+def save_json(filename: Path, data: list | dict):
     """Save JSON data to a file.
 
     Returns True on success.
     """
     data = json.dumps(data, sort_keys=True, indent=4)
-    with open(filename, "w", encoding="utf-8") as fdesc:
+    with filename.open("w", encoding="utf-8") as fdesc:
         fdesc.write(data)
         return True
     return False
```

Comment:
When can this return False?

Reply:
Yeah, returning either True or False doesn't make sense, either it returns or it throws an Exception, and the return value was never checked as well. I removed it.